src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+
+body,
+p {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
```

Comment:
Since `$font-family` and `$main-color` variables are used more than once, it's good practice to define these in a separate `_variables.scss` partial and import them into your main SCSS file. This improves maintainability and makes it easier to change these values in the future.